lib/IconPicker.lua

```diff
@@ -74,7 +74,7 @@ function IconPicker:RenderIconPicker()
     if not self.Open then return end
     self.Open, self.Draw = ImGui.Begin('Icon Picker', self.Open, ImGuiWindowFlags.None)
     if self.Draw then
-        self.Page = ImGui.InputInt("Page", self.Page, 1)
+        self.Page = ImGui.SliderInt("Page", self.Page,1, 26)

```

Comment:
Let's make this a tab list - i really don't like the input int or the slider its really non-obvious that you can move to different pages.

Reply:
I like that idea